admin-dev/themes/new-theme/js/pages/product/combination/edit.js

```diff
@@ -38,6 +39,11 @@ $(() => {
     'TextWithLengthCounter',
   ]);
 
-  new ProductSuppliersManager(ProductMap.suppliers.combinationSuppliers, false);
+  const $productForm = $(window.parent.document.querySelector(ProductMap.productForm));
+  const {eventEmitter} = window.prestashop.instance;
+  // Init product model along with input watching and syncing
+  const productFormModel = new ProductFormModel($productForm, eventEmitter);
+
+  new ProductSuppliersManager(ProductMap.suppliers.combinationSuppliers, false, productFormModel);
```

Comment:
You don't need to inject `productFormModel` here because it's only relevant for product form This needs to be refacto I'll create an issue for this
For now you can revert this modification (along with the one in suppliers manager)

Reply:
Sorry I misunderstood that this was needed to fix some js errors because I poorly handled this part ^^

I created a dedicated issue for this matter https://github.com/PrestaShop/PrestaShop/issues/26906 so it will be refactored and improved in another PR, for now, the most important is to fix the initial issue

Wer just need to add a `@todo` comment to explain it's a temporary fix and point at to the created issue